tox.ini

```diff
@@ -23,6 +23,7 @@ deps =
     jellyfish
     python-mpd2
     coverage
+    discogs_client
```

Comment:
This should probably be a hyphen, not an underscore, for consistency—but it looks like pip is happy with either one? :thinking: